prow/config/jobs/istio.yaml

```
@@ -234,6 +234,19 @@ jobs:
     - prow/config/ambient-sc.yaml
     - test.integration.ambient.kube
 
+  - name: integ-ambient-owned-cni
+    modifiers: [presubmit_optional]
+    requirements: [kind]
```

```suggestion
    optional: true
    requirements: [kind]
```

Strongly encourage setting optional until the tests are passing as expected/validated, or this will block PRs

---

Great thanks! Is this covered by presubmit_optional?